### PR TITLE
Resolve: [InfoIcon] Extract InfoIcon

### LINKED ISSUE #1765: InfoIcon
#1763 introduced an `i` icon. Extract it into an `InfoIcon` component, which accept a text for its tooltip. Later we can think about to provide different sizes (small|normal|large), too. Add a story for it.

```ts
<InfoIcon tooltip="Text for tooltip" size="normal" />
```

### PR DESCRIPTION
**Pull request by gromxyz**
Closes #1765

### CODE REVIEW

File: src/renderer/components/uielements/info/InfoIcon.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+
+import { InfoIcon } from './InfoIcon'
+
+storiesOf('Components/InfoIcon', module).add('default', () => <InfoIcon tooltip="testing info icon" />)
```
Comment (veado) [inline src/renderer/components/uielements/info/InfoIcon.stories.tsx]: Pls use syntax of latest `storybook` (some of our stories aren't updated - so you might need to check https://storybook.js.org/docs/react/writing-stories/introduction#how-to-write-stories)

**Comment by veado** [comment]
:eyes: https://github.com/thorchain/asgardex-electron/issues/1765#issuecomment-919955172

**Comment by veado** [comment]
Screen shot (swap view), pls

**Comment by gromxyz** [comment]
Screenshot 2021-09-15 at 14 40 33

**Comment by gromxyz** [comment]
Screenshot 2021-09-15 at 14 40 12
Screenshot 2021-09-15 at 14 40 33

**Review by veado — APPROVED** [review]
:+1:

**Pull request merged**